src/components/sidebar/index.js

```diff
@@ -15,11 +13,10 @@ export default function selectSidebar(
   selectedTab: TabSlug
 ): React.ComponentType<{||}> | null {
   return {
-    // calltree: CallTreeSidebar,
-    calltree: null,
+    calltree: CallTreeSidebar,
     'marker-table': null,
     'stack-chart': null,
```

Comment:
I wondered if I should enable it for the stack-chart too

Reply:
So I'm not sure how I feel about this. I think we should enable it either for all tabs or for none of them. I don't want us to hide and show the sidebar during tab switching.

Reply:
I don't feel too disturbed myself but I know what you mean. I don't have a good answer :)